chains/eclipse/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blocks:
+  confirmations: 1
+  estimateBlockTime: 0.4
```

Comment:
previously for <1s block times, we've rounded up to 1

Reply:
seems like Solana is 0.4 so happy to keep this as is tbh